src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import TodoApp from './TodoApp.js';
+import TodoAppLogin from './TodoAppLogin';
+import './App.css';
+import { 
+    BrowserRouter, 
+    Route, 
+    Redirect,
+    Switch,
+ } from 'react-router-dom';
+
+
+const isLoggedIn = () => JSON.parse(localStorage.getItem('user'));
+
+export default class App extends React.Component {
+    render() {
+        return (
+        <div className="App">
+            <header>
+                my header
+            </header>
+            <BrowserRouter>
+                <Route path='/' render={() => 
+                isLoggedIn() 
+                    ? <TodoApp />
+                    : <Redirect to='login' />
+                }/>
+                 <Route path='/login' component={TodoAppLogin} />
```

Comment:
Looks like you bring in a Switch at the top but don't sure it. Remember to wraps your routes inside the switch if you want to be able to render each component exclusively.